src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
+    public static final String OPERATION_VALUE_DELIMITER = ",";
+    public static final String SUPPLY_OPERATION_TITLE = "supply";
+    public static final String BUY_OPERATION_TITLE = "buy";
+    public static final String RESULT_OPERATION_TITLE = "result";
+    public static final int OPERATION_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        Table table = createTableWithTotals(readLinesFromFileToArray(fromFileName));
+        writeFile(composeReport(table.getSupply(), table.getBuy(), table.calculateResult()),
+                toFileName);
+    }
+
+    private String[] readLinesFromFileToArray(String filename) {
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
bad naming
never name String string, StringBUilder stringBuilder etc it wastes the opportunity to add new info, make code readable